google-translate-connector-test/src/com/axonivy/connector/googletranslate/mock/GoogleTranslateServiceMock.java

```diff
@@ -0,0 +1,82 @@
+package com.axonivy.connector.googletranslate.mock;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import javax.annotation.security.PermitAll;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.io.IOUtils;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.rest.client.config.IvyDefaultJaxRsTemplates;
+import io.swagger.v3.oas.annotations.Hidden;
+
+/**
+ * A simple REST backend for a third-party service used in tests:<br/>
+ * - provides static results and therefore simplifies our tests <br/>
+ * - does not require authentication or real-world secrets <br/>
+ * - built with ivy standard tools: easy to maintain for everyone <br/>
+ */
+@Path(GoogleTranslateServiceMock.PATH_SUFFIX)
+@PermitAll // allow unauthenticated calls
+@Hidden // do not show me on swagger-ui or openapi3 resources.
+@SuppressWarnings("all")
+public class GoogleTranslateServiceMock {
+
+  static final String PATH_SUFFIX = "mock";
```

Comment:
maybe you copied here from a bad example: but it would be better to have the service name in the resource. Otherwise there will be a service clash if you have multiple connectors in your workspace: so I'd advise to use "googleMock" instead.

Reply:
updated